style.css

```diff
@@ -127,3 +127,33 @@ nav .btn {
   text-align: center;
   max-width: none;
 }
+
+.partners-text {
```

Comment:
You could also use the `.partners a` selector for the same effect, so you wouldn't even need a class in the HTML. Unless you project to have different styles for different `<a>`, of course.